filters/__init__.py

```diff
@@ -92,3 +96,9 @@ def friendly_time(context, dt, past_="ago", future_="from now", default="Just no
             )
 
     return default
+
+
+@jinja2.contextfilter
+@filters.app_template_filter("format_time")
+def format_time(context, datetime_str):
+    return dateutil.parser.parse(datetime_str).strftime("%A, %b %d, %Y @ %I:%M %p")
```

Comment:
I love the date format! The only thing is maybe we don't need the leading zero in front of the hour. Using `%-I` instead of `%I`should get rid of the leading zero.

Reply:
Nice! I was looking for a way to do that but didn't catch that while reading the documentation. Thanks for the tip.